### ISSUE TITLE
Log error when running Indexing API

### ISSUE THREAD

**Issue by Jnyawach** [googleapis/google-api-php-client#2277]
**Hello, I scheduled a command to run google indexing on my application. The command runs fine but I receive this error:**
Declaration of Psr\Log\AbstractLogger::emergency($message, array $context = []) must be compatible with Psr\Log\LoggerInterface::emergency(Stringable|string $message, array $context = []): void {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Declaration of Psr\\Log\\AbstractLogger::emergency($message, array $context = []) must be compatible with Psr\\Log\\LoggerInterface::emergency(Stringable|string $message, array $context = []): void at /var/www/html/careermove.co.ke/public/google-api-php-client/vendor/psr/log/Psr/Log/AbstractLogger.php:22)
![image](https://user-images.githubusercontent.com/60853597/173288781-cae7ab95-95d3-4e80-8662-46461bbe8073.png)
How can I solve this? Please help.

**Comment by bshaffer** [comment]
Yes, this seems to be an issue you have with your dependencies. Both `Psr\Log\AbstractLogger` and `Psr\Log\LoggerInterface` are not maintained by this library. How those could get out of sync is beyond me... I would try removing your dependencies and reinstalling with `composer`. 

However, since the errors are happening in an installed dependency, it's not something we can help you with.
